CMakeLists.txt

```diff
@@ -722,8 +732,16 @@ if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
     include(CMakePackageConfigHelpers)
     write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/minizip-config-version.cmake
         COMPATIBILITY SameMajorVersion)
-
-    file(WRITE minizip-config.cmake.in "@PACKAGE_INIT@")
+    set(MINIZIP_CONFIG_CONTENT "@PACKAGE_INIT@")
+    if(MINIZIP_DEP_PKG)
+        string(APPEND MINIZIP_CONFIG_CONTENT "\ninclude(CMakeFindDependencyMacro)")
```

Comment:
Can you place the null-terminators `\n` at the end of the lines instead of at the beginning?

Reply:
done